CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+
+#### unreleased - upcoming
+
+- Add `requireUserWithAnyRole` and `anyRoleExcept`, to be used in the
```

Comment:
But what happens if a user has both `Analyst` and `Operator` roles - will they see the calls list or not? And how does that get decided?

I think it would be better to specify the roles that CAN see the calls list, rather than the ones that CAN'T.